.azure/templates/composer-cache.yml

```diff
@@ -0,0 +1,12 @@
+# File: simple-param.yml
+parameters:
+- name: key # name of the parameter; required
+  type: string # data type of the parameter; required
+  default: ""
+
+steps:
+  - task: Cache@2
+    inputs:
+      key: 'composer | "$(Agent.OS)" | ${{ parameters.key }} | composer.lock'
```

Comment:
when `key` is empty, would `'composer | "$(Agent.OS)" |  | composer.lock'` cause issues?

Reply:
Yes, it did. I fixed it.